MediaBrowser.Providers/Plugins/Tmdb/TmdbClientManager.cs

```diff
@@ -575,16 +576,51 @@ public void ConvertStillsToRemoteImageInfo(List<ImageData> images, string reques
         /// <param name="results">The collection to add the remote images into.</param>
         private void ConvertToRemoteImageInfo(List<ImageData> images, string size, ImageType type, string requestLanguage, List<RemoteImageInfo> results)
         {
+            int? targetHeight = null;
+            int? targetWidth = null;
+            var match = Regex.Match(size, @"(?<dimension>[hw])(?<size>[0-9]+)");
+            if (match.Success)
+            {
+                var targetSize = int.Parse(match.Groups["size"].Value, NumberStyles.Integer, CultureInfo.InvariantCulture);
+                if (string.Equals(match.Groups["dimension"].Value, "h", StringComparison.OrdinalIgnoreCase))
+                {
+                    targetHeight = targetSize;
+                }
+                else
+                {
+                    targetWidth = targetSize;
+                }
+            }
+
             for (var i = 0; i < images.Count; i++)
             {
                 var image = images[i];
+
+                int width;
+                int height;
+                if (targetHeight.HasValue)
```

Comment:
You don't have to go through all this trouble to set the dimensions. Most plugins don't even know the size in advance. You can set it to null and it will be read from the file.